### DISCUSSION TITLE
ISO install failed, system semi-functional

### DISCUSSION THREAD

**Discussion by ckindley-ambient** [Security-Onion-Solutions/securityonion#1823] [Unsupported Versions] [answered]
Hello folks,

I ran an ISO install of the latest version on a Dell PowerEdge R620 baremetal and it "encountered some errors" so never ran post-install tasks. It looks semi-successful; nothing works out of the box because the containers are firewalled off from one another. If I stop firewalld it works for a bit before iptables recreates all of its rules and breaks the system. If I then start and stop firewalld, it will work for another hour or so before iptables regenerates and breaks the system again. Attached is the setup log. Any ideas? Should I try installing on top of vanilla CentOS 7?

Thanks!

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/5490748/sosetup.log)

**Comment by weslambert** [comment]
Are you still experiencing an issue with this?

**Reply by ckindley-ambient** [reply]
Yes, I am. I was preparing a deployment on different hardware in a separate environment and ran in to the same thing! The only "error" occurs in applying the Zeek salt state:

67% - APPLYING ZEEK SALT STATE
----
[INFO    ] Loading fresh modules for state activity
[INFO    ] Fetching file from saltenv 'base', ** done ** 'zeek/init.sls'
[INFO    ] Loading fresh modules for state activity
[INFO    ] Running state [zeek_state_not_allowed] at time 19:53:09.170398
[INFO    ] Executing state test.fail_without_changes for [zeek_state_not_allowed]
[ERROR   ] Failure!
[INFO    ] Completed state [zeek_state_not_allowed] at time 19:53:09.172597 (duration_in_ms=2.2)

But it looks like test.fail_without_changes always returns a failure as it is supposed to. Both deployments were configured not to use Zeek, so this is extra weird to me. Maybe I need to try using Zeek instead? Is there a way to manually run post-installation scripts?

**Reply by dehobbs** [reply]
I'm having the same issue both in a Virtual and Physical machine. I've reinstalled easily 6 times today and have the same result every time

**Comment by jertel** [comment, marked as answer]
I've created issue https://github.com/Security-Onion-Solutions/securityonion/issues/1990 for this.

**Reply by ckindley-ambient** [reply]
Thanks jertel. I appreciate the help as I am a GitHub noob.
